namespace java thrift.test

struct mySetStruct {
  1: set<i16> a
}
